src/services/healthCheckByTime.ts

```diff
@@ -0,0 +1,24 @@
+import { Pool } from "pg";
+
+import { UtilEither} from "../utils";
+
+export interface CardanoFrag {
+  behindby: {
+    years?: number,
+    months?: number,
+    days?: number,
+    hours?: number,
+    minutes?: number,
+    seconds?: number,
+  };
+}
+
+export const askBehindBy = async (pool: Pool) : Promise<UtilEither<CardanoFrag>> => {
+  const query = `
+  select now() - max(time) as behindBy from block;
```

Comment:
I didn't know it would give the nice split in seconds, milli, minutes, hours... brilliant.